app/models.py

```diff
@@ -211,6 +211,7 @@ def serialize(self):
             'supplierId': self.supplier.supplier_id,
             'supplierName': self.supplier.name,
             'frameworkName': self.framework.name,
+            'updatedAt': self.updated_at.strftime("%Y-%m-%dT%H:%M:%S%Z"),
```

Comment:
`updatedAt` in the `User` model doesn't explicitly format the timestamp. We should be consistent. Also, there is [`datetime.isoformat`](https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat)